snapcraft.yaml

```diff
@@ -66,6 +66,11 @@ apps:
     restart-condition: always
     plugs: [network, network-bind]
 
+  cron-lite:
```

Comment:
Remember the snap contains everything needed by Nextcloud (mysql, apache, etc.). With that in mind, "cron-lite" is not the most descriptive of names. How about "nextcloud-cron"?